/**:
  ros__parameters:

    max_speed: 0.50      # Начальная максимальная скорость робота
    interIn_speed: 0.38  # Скорость после знака перекрестка
    inter_speed: 0.35    # Скорость при движении по перекрестку
    interOut_speed: 0.40 # Скорость после выезда из перекрестка

    center_offset: 20.0   # Начальное смещение центра от центра полосы
    interIn_offset: 40.0  # Смещение после знака перекрестка
    interL_offset: 80.0   # Смещение при повороте налево
    interR_offset: 20.0   # Смещение при повороте направо
    interOut_offset: 40.0 # Смещение после знака перекрестка

    # Размера буфера с ошибками
    max_len: 50 

    # PID константы для угла поворота  
    Kp_ang: 0.004
    Ki_ang: 0.00002
    Kd_ang: 0.0

    # PID константы для скорости
    Kp_vel: 0.0007
    Ki_vel: 0.000005
    Kd_vel: 0.0

    # Минимальная площадь bbox для детекции знака
    min_square: 6000

    # Количество детекций для усреднения ответа
    detects_cnt: 5